### DISCUSSION TITLE
Add Unit Column in PDF Design

### DISCUSSION THREAD

**Discussion by JEEVAN28** [YetiForceCompany/YetiForceCRM#14144] [Wishlist]
Hello,

I am trying to add Unit Column in /www/wwwroot/default/crm/yf/app/TextParser/ProductsTableImages.php around line 49,

foreach (['ItemNumber', 'Name', 'Quantity', **'Unit'**, 'UnitPrice', 'TotalPrice'] as $fieldType) { 

same does not display in PDF output.
![image](https://user-images.githubusercontent.com/12723595/106363997-f0e56b80-6351-11eb-92dd-bee43ff3891f.png)

kindly assist.

Regards
Jeevan Pendhari

**Comment by davide-alghi** [comment]
Hi @JEEVAN28,
because the 'Unit' field has got wrong type in database: 'Value'.

https://github.com/YetiForceCompany/YetiForceCRM/pull/12499

**Comment by hahyes** [comment]
> Hi @JEEVAN28,
> because the 'Unit' field has got wrong type in database: 'Value'.
> 
> #12499

Is this bugfix in Service Pack v6 for YetiForce 6.1?

**Comment by davide-alghi** [comment]
Hi @hahyes,
I just installed "YetiForce CRM 6.1 Service Pack v6" and fix is not there. 
Unit and Subunit type fixing, in database, also requires changing some text-parser PDF functions.
Perhaps in next updates/upgrades, YF Team will provide these changes.

**Reply by mariuszkrzaczkowski** [reply]
> Unit and Subunit type fixing, in database, also requires changing some text-parser PDF functions.

Does not require,  try this on gitdev: 

$(inventory : type=table columns=seq,name,qty,unit,subunit,price,total,net href=no)$

**Reply by davide-alghi** [reply]
tnx @mariuszkrzaczkowski
